app/src/main/java/fr/neamar/kiss/DataHandler.java

```diff
@@ -77,7 +77,7 @@ public DataHandler(Context context) {
 
     @Override
     public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, String key) {
-        if (key.startsWith("enable-")) {
+        if (key.startsWith("enable-" )) {
```

Comment:
I didn't do it on purpose. After "Reformat code" shortcut android studio generated them.